diff --git a/node_modules/element-react/dist/npm/es5/src/upload/index.js b/node_modules/element-react/dist/npm/es5/src/upload/index.js
index 9c9ed47..8d7eab2 100644
--- a/node_modules/element-react/dist/npm/es5/src/upload/index.js
+++ b/node_modules/element-react/dist/npm/es5/src/upload/index.js
@@ -1,7 +1,6 @@
 'use strict';

 var _react = require('react');
-var _reactHotLoader = require('react-hot-loader');
 var _uploadList = require('./uploadList');
 var _upload = require('./upload');
 var _upload2 = _interopRequireDefault(_upload);
@@ -11,8 +10,6 @@ var _upload2 = _interopRequireDefault(_upload);
 var _i18n = require('../i18n');
 var _i18n2 = _interopRequireDefault(_i18n);

-var _reactHotLoader2 = _interopRequireDefault(_reactHotLoader);
-
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

 var _class, _temp;
@@ -193,5 +190,3 @@ _temp = _class, _class.contextTypes = {
 }, _temp));

 module.exports = Upload;
-
-_reactHotLoader2.default.register(Upload, 'Upload', 'src/upload/index.js');
